controllers/operands/cdi.go

```diff
@@ -17,6 +17,7 @@ import (
 
 const (
 	HonorWaitForFirstConsumerGate = "HonorWaitForFirstConsumer"
+	DataVolumeClaimAdoptionGate   = "DataVolumeClaimAdoption"
```

Comment:
If this const is not used outside of the `operands` package, let's not expose it.